core/src/main/java/org/apache/calcite/rel/rules/ReduceDecimalsRule.java

```diff
@@ -152,13 +152,6 @@ public RexNode visitCall(RexCall call) {
       }
 
       // permanently updates a call in place
```

Comment:
Should the comment be removed?

Reply:
Yes, I update the PR. Thanks for reminding me.